protocols/perf/src/lib.rs

```diff
@@ -35,6 +35,44 @@ pub mod server;
 
 pub const PROTOCOL_NAME: StreamProtocol = StreamProtocol::new("/perf/1.0.0");
 
+#[derive(Debug, Clone, Copy)]
+pub enum RunUpdate {
+    Progressed(Progressed),
+    Finished(Finished),
```

Comment:
The spec calls them `intermediate` and `final`, right? Maybe stick to that terminology here too?

Reply:
:+1: